### DISCUSSION TITLE
씬 이동후에 캐릭터를 이동된 씬에 어떻게 로드하나요?

### DISCUSSION THREAD

**Discussion by kyc3550** [naverz/zepeto-studio-kor#674] [5. Q&A]
이동 할 씬에 gameobject를 생성하면 메인 씬에서 제 캐릭터가 두개 로드 되는데 씬 생성시 최초 실행되는 함수가 있나요?

**Comment by pumfit** [comment]
제목에 대한 답을 하면 기존 캐릭터 생성 로직과 같이 이동한 씬에서도 캐릭터를 생성해주면 이동된 씬에서도 정상적으로 로드 됩니다. 다만 질문 내용에 대해서는 실행하신 코드가 있다면 더 자세히 알 수 있을 것 같습니다!
